streaming/vision/base.py

```diff
@@ -27,11 +29,29 @@ def __call__(self, x: Any, y: Any) -> Tuple[Any, Any]:
 
 
 class VisionDataset(Dataset):
+    """
+    Base Class for creating a Vision streaming datasets. It is recommend to override the ``__getitem__`` method.
+
+    Args:
+        local (str): Local filesystem directory where dataset is cached during operation.
+        split (str): The dataset split to use, either 'train' or 'val'.
+        remote (str, optional): Remote directory (S3 or local filesystem) where dataset is stored. Default: ``None``.
+        shuffle (bool, optional): Whether to shuffle the train samples in this dataset. Default: ``True``.
+        transforms (callable, optional): A function/transforms that takes in an image and a label and returns the transformed versions of both. Default: ``None``.
```

Comment:
< 100 col

Reply:
I rearranged it for now manually but we need some automation to check the doc max length. I will look into it in next PR.